From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: dkurepa <dkurepa@microsoft.com>
Date: Thu, 22 Dec 2022 15:58:06 +0100
Subject: [PATCH] Test patch

---
 Directory.Build.props | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Directory.Build.props b/Directory.Build.props
index 1353696a29..0ed0eb4b9d 100644
--- a/Directory.Build.props
+++ b/Directory.Build.props
@@ -1,5 +1,6 @@
 <!-- Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the MIT license. See License.txt in the project root for full license information. -->
 <Project>
+SOMETHING SHOULD BE WRITTEN HERE
   <Import Project="Sdk.props" Sdk="Microsoft.DotNet.Arcade.Sdk" />
 
   <PropertyGroup>
